src/components/Modal/Modal.scss

```diff
@@ -0,0 +1,103 @@
+@import "../../styles/mixins";
+
+.modal-backdrop {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.7);
+  z-index: 1000;
+  gap: 8px;
+
+  .modal-btn-container {
+    width: 573px;
```

Comment:
the hard coded width is not good for the responsive , try to play with the responsive option in the browser and u'll see whats happen